cli/flags.rs

```diff
@@ -119,7 +121,7 @@ pub enum DenoSubcommand {
 
 impl Default for DenoSubcommand {
   fn default() -> DenoSubcommand {
-    DenoSubcommand::Repl
+    DenoSubcommand::Repl { eval: None }
```

Comment:
Note that `deno --eval "console.log(5);"` does not work with this PR. The `repl` subcommand is required. Should that change?

Reply:
I also think it's not worth it